src/Altinn.AccessManagement.Core/Services/AppsInstanceDelegationService.cs

```diff
@@ -359,14 +365,70 @@ private async Task<List<InstanceRightDelegationResult>> DelegateRights(InstanceR
     }
 
     /// <inheritdoc/>
-    public Task<Result<bool>> Get(CancellationToken cancellationToken = default)
+    public async Task<Result<List<AppsInstanceDelegationResponse>>> Get(AppsInstanceGetRequest request, CancellationToken cancellationToken = default)
     {
-        throw new NotImplementedException();
+        ValidationErrorBuilder errors = default;
+
+        // Fetch rights valid for delegation
+        ServiceResource resource = (await _resourceRegistryClient.GetResourceList(cancellationToken)).Find(r => r.Identifier == request.ResourceId);
+        List<Right> delegableRights = null;
+
+        if (resource == null)
+        {
+            errors.Add(ValidationErrors.InvalidResource, "request.Resource");
+        }
+        else
+        {
+            RightsQuery rightsQueryForApp = new RightsQuery
+            {
+                Type = RightsQueryType.AltinnApp,
+                To = new AttributeMatch { Id = AltinnXacmlConstants.MatchAttributeIdentifiers.ResourceDelegationAttribute, Value = request.PerformingResourceId.ValueSpan.ToString() }.SingleToList(),
+                From = resource.AuthorizationReference,
+                Resource = resource
+            };
+
+            try
+            {
+                delegableRights = await _pip.GetDelegableRightsByApp(rightsQueryForApp, cancellationToken);
+            }
+            catch (ValidationException)
+            {
+                errors.Add(ValidationErrors.MissingPolicy, "request.Resource");
+            }
+
+            if (delegableRights == null || !delegableRights.Exists(r => r.CanDelegate.HasValue && r.CanDelegate.Value))
```

Comment:
Burde kanskje dokumentert/kommentert på at dette i praksis er Autorisasjonssjekken på at autentisert app har minst _en_ tilgang for å i det hele tatt få lov til å gjøre get oppslaget